src/main/java/org/dependencytrack/tasks/VulnerabilityAnalysisTask.java

```diff
@@ -166,7 +172,7 @@ private void performAnalysis(final Subscriber scanTask, final VulnerabilityAnaly
             }
             // Clear the transient cache result for each component.
             // Each analyzer will have its own result. Therefore, we do not want to mix them.
-            event.getComponents().forEach(c -> c.setCacheResult(null));
+            event.getComponents().parallelStream().forEach(c -> c.setCacheResult(null));
```

Comment:
I don't think this is a good idea. See https://stackoverflow.com/a/20375622

Reply:
this was not intended to be sent in PR before. I will revert this

Reply:
removed